fixit/common/tests/test_config.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+from pathlib import Path
+
+from jsonschema.exceptions import ValidationError
+from libcst.testing.utils import UnitTest
+
+from fixit.common.config import get_validated_settings
+
+
+class TestConfig(UnitTest):
+    def test_validated_settings_with_bad_types(self) -> None:
+        bad_config = {"block_list_rules": False}
+        with self.assertRaises(ValidationError) as ex:
+            get_validated_settings(bad_config, Path("."))
+        self.assertIn("False is not of type 'array'", str(ex.exception))
```

Comment:
The assertRaises and assertIn calls can be combined with a assertRaisesRegex to be simpler and more readable.
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex

Reply:
Good call!